internal/common/rules/clienttools.go

```diff
@@ -0,0 +1,174 @@
+package rules
```

Comment:
These are for the clients to use right? Great addition!

Reply:
Yup